### DISCUSSION TITLE
What does the vim mode do and how do you you it

### DISCUSSION THREAD

**Discussion by CapedBojji** [sammy-ette/Hilbish#307] [General]
I am trying to figure out what the vim mode does but the documentation is very confusing

**Comment by sammy-ette** [comment]
[vim mode](https://rosettea.github.io/Hilbish/docs/api/hilbish/#:~:text=the%20login%20shell%3F-,vimMode,-Current%20Vim%20input) is an alternative input mode which allows using vim-like keybinds to insert, delete, etc. instead of using emacs style binds (ctrl-d, whatever)

the [vimMode property](https://rosettea.github.io/Hilbish/docs/api/hilbish/#:~:text=the%20login%20shell%3F-,vimMode,-Current%20Vim%20input) will hold the current vim mode and can be used in the prompt.
